src/generic/activityindicator.cpp

```diff
@@ -123,6 +123,7 @@ class wxActivityIndicatorImpl
     {
         wxPaintDC pdc(m_win);
 
+#if wxUSE_GRAPHICS_CONTEXT
```

Comment:
Hmm, we should probably disable `wxUSE_ACTIVITYINDICATOR` in this case in `wx/chkconf.h`, as it's really not much use if it can't draw itself.

Reply:
You mean in addition to the `if wxUSE_GRAPHICS_CONTEXT` here? Or should I remove this change and make the other one?

Reply:
I think the check in `wx/chkconf.h` would be enough. BTW, even though the check could be skipped for wxGTK (as it has a native version), I don't think it's worth doing it, as wxGC is always available in wxGTK3 (as it's used for wxDC) and I don't think we care enough about GTK+ between 2.20 and 3 to handle it specially...